website/static/js/registrationUtils.js

```diff
@@ -248,16 +248,34 @@ var Draft = function(params, metaSchema) {
 };
 Draft.prototype.preRegisterPrompts = function(response, confirm) {
     var self = this;
-    bootbox.confirm({
+    var ViewModel = registrationEmbargo.ViewModel;
+    var viewModel = new ViewModel();
+    viewModel.canRegister = ko.computed(function() {
```

Comment:
It seems like we're adding a lot of code to an embargo view model, but we're not actually adding it to the viewModel itself: we're changing one instance in `registrationUtils.js` instead of the definition in `registrationEmbargo.js`. 

I'd be curious to know more about the reason for the choice. Preference is to keep related code for the behavior of a given viewmodel together if possible.